app/javascript/dashboard/assets/scss/_utility-helpers.scss

```diff
@@ -10,8 +10,18 @@
   margin-right: var(--space-smaller);
 }
 
+.margin-left-smaller {
```

Comment:
We have `margin-1` equivalent available in https://get.foundation/sites/docs/prototyping-utilities.html; please use it.

Reply:
Done https://github.com/chatwoot/chatwoot/pull/6369/commits/158dc29409d3099dfb8e4268cfad6e9447e76fe3